dashboard/db/migrate/20161111020106_create_pd_enrollment_notifications.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePdEnrollmentNotifications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_enrollment_notifications do |t|
+      t.timestamps
+      t.belongs_to :pd_enrollment, null: false
+
+      # The name of notification: i.e. name of action in Pd::WorkshopMailer
+      t.string :name
```

Comment:
good point. if "action name" will always be accurate and more specific then it seems like an improvement, but if you're not sure then "name" seems fine (especially if you choose to add a comment). I agree that "notification name" doesn't help because it's kind of already implied by the name of the table.